### PR TITLE
Add a new message type: JogJoint.

### PR DESCRIPTION
**Pull request by AndyZe**
This is related to real-time teleoperation of the robot, MoveIt! PR [#1244](https://github.com/ros-planning/moveit/pull/1244)

### CODE REVIEW

File: msg/JogJoint.msg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ # Used in time-stamping the message.
+ Header header
+ 
+ # Name list of the joints. You don't need to specify all joints of the
+ # robot. Joint names are case-sensitive.
+ string[] joint_names
+ 
+ # Relative displacement of the joints to jog. The order must be
```
Comment (bmagyar) [inline msg/JogJoint.msg]: "Relative displacement of the joints to jog" --> "Relative displacement of the joints listed in `joint_names`, the order must be identical."

File: msg/JogJoint.msg [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+ # identical to joint_names. Unit is radian for revolute joints,
+ # meter for linear joints.
```
Comment (bmagyar) [inline msg/JogJoint.msg]: This assumes that the listed joints all operate in position mode. A mobile base wheel or other continuous joints are often able to handle both position and velocity requests. This is not always the case, please add a comment clarifying this.

**Comment by rhaschke** [comment]
I don't have a particularly strong opinion on adding this msg neither.
+1 for considering control_msgs.

**Comment by AndyZe** [comment]
OK, I'm making those changes and moving this to control_msgs.

**Comment by AndyZe** [comment]
See https://github.com/ros-controls/control_msgs/pull/26

**Pull request closed without merging**